base_local_planner/src/local_planner_limits/__init__.py

```diff
@@ -24,6 +24,7 @@ def add_generic_localplanner_params(gen):
     gen.add("min_vel_y", double_t, 0, "The minimum y velocity for the robot in m/s", -0.1)
 
     gen.add("max_vel_theta", double_t, 0, "The absolute value of the maximum rotational velocity for the robot in rad/s",  1.0, 0)
+    gen.add("max_vel_theta_spin", double_t, 0, "The absolute value of the maximum rotational velocity for the robot in rad/s when spinning in place. Should be greater than or equal to max_vel_theta",  1.0, 0)
```

Comment:
Q: technically, it would work correctly even if `max_vel_theta_spin` is smaller than `max_vel_theta` if we just change 
`double max_vel_th_spin = std::max(limits->max_vel_theta_spin, max_vel_th);` below to
`double max_vel_th_spin = limits->max_vel_theta_spin;`

Reply:
I guess, but we're not technically checking for spin in place anymore. We're just saying if the robot has low lin vel, then we allow for higher ang vel. I don't think it makes much sense to do the opposite, hence why I'm using the max.